examples/diagnostic_protocol/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
-
+project(diagnostic_protocol_example)
```

Comment:
To make the examples build independent, I added a project command to each of em.